packages/typespec-azure-resource-manager/test/rules/no-response-body.test.ts

```diff
@@ -33,6 +33,34 @@ describe("typespec-azure-resource-manager: no response body rule", () => {
       .toBeValid();
   });
 
+  it("is valid if 204 responses have no body", async () => {
```

Comment:
I think we need a test to verify that 200 responses with no body are valid in head, delete, and post operations

Reply:
If we already have these tests, then this looks good to merge